### ISSUE TITLE
Crash on load-chat: `attempt to index global 'bit32' (a nil value)`

### ISSUE THREAD

**Issue by tsweet64** [BanchouBoo/mpv-youtube-chat#2]
Hello and thank you for this awesome project. However, on my system, upon attempting to `load-chat`, the script crashes with the following error:

```
[mpv_youtube_chat] 
[mpv_youtube_chat] stack traceback:
[mpv_youtube_chat]      /home/<name>/.config/mpv/scripts/mpv-youtube-chat/main.lua:330: in function 'load_live_chat'
[mpv_youtube_chat]      /home/<name>/.config/mpv/scripts/mpv-youtube-chat/main.lua:388: in function 'handler'
[mpv_youtube_chat]      mp.defaults:380: in function 'handler'
[mpv_youtube_chat]      mp.defaults:510: in function 'call_event_handlers'
[mpv_youtube_chat]      mp.defaults:552: in function 'dispatch_events'
[mpv_youtube_chat]      mp.defaults:503: in function <mp.defaults:502>
[mpv_youtube_chat]      [C]: at 0x5623556aa5e0
[mpv_youtube_chat]      [C]: at 0x5623556aaaa0
[mpv_youtube_chat] Lua error: /home/<name>/.config/mpv/scripts/mpv-youtube-chat/main.lua:242: attempt to index global 'bit32' (a nil value)
```

I have the following in input.conf and the error occurs when pressing ctrl+shift+c:
```
CTRL+Shift+c script-message load-chat
Alt+CTRL+Shift+c script-message unload-chat
```

`mpv --version`:
```
mpv 0.35.1-dirty Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects
 built on Fri Apr 21 07:11:25 2023
FFmpeg library versions:
   libavutil       58.2.100
   libavcodec      60.3.100
   libavformat     60.3.100
   libswscale      7.1.100
   libavfilter     9.3.100
   libswresample   4.10.100
FFmpeg version: n6.0
```

Arch Linux, lua verison `5.4.6`, latest git master of `mpv-youtube-chat`, other scripts installed are `mpvacious` and `autosubsync-mpv`.

Any ideas how to troubleshoot this problem?

**Comment by BanchouBoo** [comment]
The script was written with lua 5.2 in mind, which doesn't have operators for bitwise operations like 5.4. I'm unsure if there's a simple way to support both, I don't really feel like looking into it right now but maybe I'll get to it soon, otherwise if somebody wants to PR a fix I can merge that.

**Comment by BanchouBoo** [comment]
Closed by #3
